### PR TITLE
Test on scala 2.12 & 2.11.

### PR DESCRIPTION
**Pull request by kshakir**
Bumped scala and sbt to latest versions (that have full library support).
Removed unused library exclusions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 sudo: false
 language: scala
 scala:
-  - 2.11.8
+  - 2.12.3
```
Comment (mcovarr) [inline .travis.yml:4]: oh fun we had mismatched Scala versions between Travis and local 😦

**Review by mcovarr — APPROVED** [review]

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/lenthall/pull-request/44/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/lenthall)

**Review by danbills — APPROVED** [review]

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/lenthall/pull-request/44/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/lenthall)

**Comment by kshakir** [comment]
I think pullapprove wants a recheck after rebasing on the bump to v2. Ignoring and merging.

**Pull request merged**